### PR TITLE
Puppet RHEL6 Support

### PR DESCRIPTION
**Pull request by jyejare**
Hi @JacobCallahan ,

Adding support for rhel6 puppet client.

I tested this on my system and client registers fine. But somehow the subscription status is unentitled.

Please check and merge.

### CODE REVIEW

File: RHEL6/startup.sh [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+sed -i "2i\    server = $SATHOST" /etc/puppet/puppet.conf
+echo "    ca_server = $SATHOST" >> /etc/puppet/puppet.conf
+echo "    server = $SATHOST" >> /etc/puppet/puppet.conf
+echo "    environment = production" >> /etc/puppet/puppet.conf
+echo "    pluginsync = true" >> /etc/puppet/puppet.conf
+echo "    report = true" >> /etc/puppet/puppet.conf
+echo "    ignoreschedules = true" >> /etc/puppet/puppet.conf
+echo "    daemon = false" >> /etc/puppet/puppet.conf
+
+echo "10.19.34.24 yttrium.idmqe.lab.eng.bos.redhat.com yttrium" >> /etc/hosts
```
Comment (jyejare) [inline RHEL6/startup.sh:50]: @JacobCallahan , Also please check if this needs to be updated.

**Comment by JacobCallahan** [comment]
@jyejare are you rebased to the latest on master branch?

Also, this shouldn't go in the RHEL6 directory, so you should create a new one called RHEL6-Puppet and put the contents in there.

File: RHEL6/startup.sh (hunk 1 above)
Comment (JacobCallahan) [reply RHEL6/startup.sh:50]: why is this line needed?

Comment (jyejare) [reply RHEL6/startup.sh:50]: @JacobCallahan , I thought it is required to set hostname . If it is not, I will remove.

**Comment by jyejare** [comment]
@JacobCallahan , I have created 2 directories in **puppet branch** as you do in master for RHEL7 and RHEL6. So do you want to rename both of them to RHEL7_puppet and RHEL6_puppet. I dont think so as the branch itself is completely different.

**Comment by JacobCallahan** [comment]
@jyejare yes, please do rename them, as all non-master branches will be going away. We're going to try to keep everything on master now.
Thanks!

File: RHEL6/startup.sh (hunk 1 above)
Comment (JacobCallahan) [reply RHEL6/startup.sh:50]: If it is required to set the hostname, that should be handled using the SATHOST environment variable.

**Pull request closed without merging**